src/app/workflow/register-workflow-modal/register-github-app-modal/register-github-app-modal.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="mb-3" *ngIf="Dockstore.FEATURES.enableNotebooks && entryType === EntryType.Notebook">
```

Comment:
This dialog should have a title, like Register Workflow

Reply:
Thank you!!